app/subsystems/tasks/assistants/i_reading_assistant.rb

```
@@ -169,9 +169,10 @@ def self.tasked_exercise(exercise_fragment:,
     exercises = Content::Routines::SearchExercises[
                   tag: exercise_fragment.embed_tag
                 ]
-    exercise = exercises.first
-    TaskExercise[exercise: exercises.first, title: title,
-                 can_be_recovered: can_be_recovered, task_step: step]
+    if exercise = exercises.first
+      TaskExercise[exercise: exercise, title: title,
```

Is this causing a silent failure when our input data is badly-formed?  Why would `exercises` be legitimately empty?